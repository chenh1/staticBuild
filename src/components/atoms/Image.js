import React, {PropTypes} from 'react';

const Image = ({imageSrc, alt}) => {
    return (
        <img src={imageSrc} alt={alt}/>
    );
};

Image.propTypes = {
    imageSrc: PropTypes.string.isRequired,
    alt: PropTypes.string.isRequired
};

export default Image;