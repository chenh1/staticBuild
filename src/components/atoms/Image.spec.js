import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import Image from './Image';

test('renders img tag with correct attributes', () => {
    const tree = renderer.create(<Image
        src="/mockSrc"
        alt="Mock Image" />
    ).toJSON();

    expect(tree).toMatchSnapshot();
});
