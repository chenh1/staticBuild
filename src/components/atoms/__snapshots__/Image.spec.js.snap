// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders img tag with correct attributes 1`] = `
<img
  alt="Mock Image"
  src={undefined}
/>
`;
